spec/models/conference_spec.rb

```diff
@@ -41,6 +41,16 @@
       end
     end
   end
+  
+  describe '#dates' do
+    subject { FactoryGirl.build_stubbed(:conference) }
+    
+    it 'has a date range' do
```

Comment:
Not sure if the expectations here really fit the test. It does not test whether or not the return value is a `DateRange` object. Whether or not the internal values of the `DateRange` object match those of the conference is not really relevant I would say - that's internal logic of the `DateRange` class.

You could test the call to the constructor:
```ruby
it "has a date range" do
  expect(DateRange)
    .to receive(:new)
    .with(start_date: subject.starts_on, ends_on: subject.ends_on)
  expect(subject.dates).to be_a(DateRange)
end
```